spec/services/filter_application_choices_for_providers_spec.rb

```diff
@@ -80,7 +80,7 @@
         application_choices.last.update(status: :inactive)
         result = described_class.call(application_choices:, filters: { status: ['awaiting_provider_decision'] })
 
-        expect(result).to eq([application_choices.first, application_choices.last])
+        expect(result.sort).to eq([application_choices.first, application_choices.last].sort)
```

Comment:
If we don't care about ordering by `status` we can use [`match_array`](https://rspec.info/documentation/3.12/rspec-expectations/#built-in-matchers)

Reply:
Yeah, didn't put much thought into it. Fixed now.

Reply:
But the query is not sorting by status. This will continue to be flakey because the order returned by the query is not determinate. We should use `match_array`.

Reply:
Sorry, I didn't actually look at the docs for `contain_exactly`.